### ISSUE TITLE
fix `pyroute2.arp` export

### ISSUE THREAD

**Issue by svinota** [svinota/pyroute2#895]
With what do I replace `pyroute2.arp` ? I have a module that contains, amongst other stuff,
```
import pyroute2
import pyroute2.arp
...
        self.ip = pyroute2.IPRoute()
...
        for ifl in self.ip.get_links():
            ifl_index = ifl["index"]
            ifl_type = pyroute2.arp.ARPHRD_VALUES.get(ifl["ifi_type"], "UNKNOWN")
```
and I get an error on importing:
```
mymodule.py:6: in <module>
    import pyroute2.arp
E   ModuleNotFoundError: No module named 'pyroute2.arp'
```
Also, it seems that python 3.5 support has been dropped?

_Originally posted by @oseiberts11 in https://github.com/svinota/pyroute2/discussions/786#discussioncomment-2558030_

**Comment by svinota** [comment]
@oseiberts11 The fix for that is pushed into the master branch.

If the git version is not an option, you can use `import pr2modules.arp` until the next release will be tagged and rolled out.

Regarding Python 3.5: yes, I don't test the project on this version anymore.

**Comment by svinota** [comment]
Closing as resolved. Will you find any further issue — don't hesitate to reopen or open a new ticket. Thanks!

**Comment by 257** [comment]
> Closing as resolved. Will you find any further issue — don't hesitate to reopen or open a new ticket. Thanks!

please re-open, thanks.

**Comment by 257** [comment]
> @257 could be an upgrade to >=0.7.1 an option?

not really, unless i get distro's (opensuse) package maintainer to bump. will try.

**Comment by svinota** [comment]
That's a bug in the packaging — at least on my side. The `pyroute2` pypi is missing the dependency on `pyroute2.protocols`.

Either OpenSuse has `pyroute2.protocols` too — and it's install will help — or it's missing it completely, and then only the version bump may help, as >=0.7.1 doesn't use this packaging scheme anymore.
